app/api/models/errors/CommonMtdErrors.scala

```diff
@@ -36,8 +36,6 @@ object ValueFormatError extends MtdError("FORMAT_VALUE", "The value must be betw
 
 }
 
-object CheckpointIdFormatError extends MtdError("FORMAT_CHECKPOINT_ID", "The provided checkpoint ID is invalid", BAD_REQUEST)
```

Comment:
Could we maybe remove the unused `SubmissionIdNotFoundError` and `ServiceUnavailableError`?

Reply:
These common errors are just those that _might_ be used and hopefully keeps things consistent across APIs (even this test one). If we ever create a MTD bootstrap library I would expect this file to be in there.